apps/studio/pages/project/[ref]/database/backups/restore-to-new-project.tsx

```diff
@@ -0,0 +1,510 @@
+  const [selectedBackupId, setSelectedBackupId] = useState<number | null>(null)
+  const [showConfirmationDialog, setShowConfirmationDialog] = useState(false)
+  const [showNewProjectDialog, setShowNewProjectDialog] = useState(false)
+
+  const isLoading = isPermissionsLoaded && cloneBackupsLoading && backupsLoading
+
+  const { mutate: triggerClone } = useProjectCloneMutation({
+    onError: (error) => {
+      console.error('error', error)
+      toast.error('Failed to restore to new project')
+    },
+    onSuccess: () => {
+      toast.success('Restoration process started')
+    },
+  })
+
+  const FormSchema = z.object({
+    name: z.string().min(1),
+    password: z.string().min(1),
+  })
+
+  const form = useForm<z.infer<typeof FormSchema>>({
+    resolver: zodResolver(FormSchema),
+    defaultValues: {
+      name: '',
+      password: '',
+    },
+  })
+
+  const [passwordStrengthScore, setPasswordStrengthScore] = useState(0)
+  const [passwordStrengthMessage, setPasswordStrengthMessage] = useState('')
+
+  const delayedCheckPasswordStrength = useRef(
+    debounce((value: string) => checkPasswordStrength(value), 300)
+  ).current
+
+  async function checkPasswordStrength(value: string) {
+    const { message, strength } = await passwordStrength(value)
+    setPasswordStrengthScore(strength)
+    setPasswordStrengthMessage(message)
+  }
+
+  function generateStrongPassword() {
+    const password = generator.generate({
+      length: 16,
+      numbers: true,
+      uppercase: true,
+    })
+
+    form.setValue('password', password)
+    delayedCheckPasswordStrength(password)
+  }
+
+  if (isLoading) {
+    return <GenericSkeletonLoader />
+  }
+
+  if (!canReadPhysicalBackups) {
+    return <NoPermission resourceText="view backups" />
+  }
+
+  if (!canTriggerPhysicalBackups) {
+    return <NoPermission resourceText="restore backups" />
+  }
+
+  if (!IS_PG15_OR_ABOVE) {
+    return (
+      <Admonition
+        type="default"
+        title="Restore to new project is not available for this database version"
+        description={
+          <>
+            Restore to new project is only available for Postgres 15 and above.
+            <br /> Go to{' '}
+            <Link className="underline" href={`/project/${project?.ref}/settings/infrastructure`}>
+              infrastructure settings
+            </Link>{' '}
+            to upgrade your database version.
+          </>
+        }
+      />
+    )
+  }
+
+  if (!PHYSICAL_BACKUPS_ENABLED) {
+    return (
+      <Admonition
+        type="default"
+        title="Restore to new project requires physical backups"
+        description={
+          <>
+            Enable physical backups to restore to a new project.
+            <br /> Find out more about how backups work at supabase{' '}
+            <Link
+              target="_blank"
+              className="underline"
+              href="https://supabase.com/docs/guides/platform/backups"
```

Comment:
this docs doesnt tell me much about restoring to a new project

are we going to have docs on that specifically?